src/sentry/api/endpoints/organization_eventid.py

```diff
@@ -14,6 +15,7 @@
 
 @region_silo_endpoint
 class EventIdLookupEndpoint(OrganizationEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```

Comment:
```suggestion
    owner = ApiOwner.ISSUES
```

This endpoint is used to find issues for event ids.